ComAmazonawsDynamodb/runtimes/java/src/main/smithy-generated/software/amazon/cryptography/services/dynamodb/internaldafny/ToDafny.java

```diff
@@ -10489,9 +10489,11 @@ public static TimeToLiveStatus TimeToLiveStatus(String nativeValue) {
   }
 
   public static Error Error(DynamoDbException nativeValue) {
-    // BEGIN MANUAL EDIT
-    return new Error_Opaque(nativeValue);
-    // END MANUAL EDIT
+    return Error.create_Opaque(nativeValue);
```

Comment:
What's the difference between `Error_Opaque` vs `Error.create_Opaque`?

Reply:
Dafny would prefer we always use their constructor methods instead of the class construction methods they accidentally publish.
Eventually, in a major breaking change, Dafny will make the `new` constructors private.
Before that happens, Smithy-Dafny will need to be corrected.

But that is all for another day.

For Today, when I modify Smtihy-Dafny, I generally try to correct this.
Logically, the methods are identical.